File: v131016/TeeNew/Circled_Gradient.pas
unit Circled_Gradient;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QStdCtrls, QExtCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, StdCtrls, ExtCtrls,
  {$ENDIF}
  Base, TeEngine, Series, TeePolar, TeeProcs, Chart;

type
  TCircledGradient = class(TBaseForm)
    Series1: TPolarSeries;
    Button1: TButton;
    procedure Button1Click(Sender: TObject);
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.dfm}
{$ELSE}
{$R *.xfm}
{$ENDIF}

uses TeeEdiGrad;

procedure TCircledGradient.Button1Click(Sender: TObject);
begin
  EditTeeGradient(Self,Series1.CircleGradient);
end;

procedure TCircledGradient.FormCreate(Sender: TObject);
begin
  inherited;
  Series1.FillSampleValues;
  Series1.Shadow.Visible := false;
end;

initialization
  RegisterClass(TCircledGradient);
end.

File: TeeNew/Builder2007/Circled_Gradient.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Circled_Gradient.h"
#include "TeeEdiGrad.hpp"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Base"
#pragma link "TeePolar"
#pragma resource "*.dfm"
TCircledGradient *CircledGradient;
//---------------------------------------------------------------------------
__fastcall TCircledGradient::TCircledGradient(TComponent* Owner)
  : TBaseForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TCircledGradient::Button1Click(TObject *Sender)
{
  EditTeeGradient(this,Series1->CircleGradient);
}
//---------------------------------------------------------------------------

void __fastcall TCircledGradient::FormCreate(TObject *Sender)
{
  Series1->FillSampleValues(10);
}
//---------------------------------------------------------------------------

